lib/susedistribution.pm

```diff
@@ -3,7 +3,7 @@ use base 'distribution';
 use serial_terminal ();
 use strict;
 use utils
-  qw(type_string_slow ensure_unlocked_desktop save_svirt_pty get_root_console_tty get_x11_console_tty ensure_serialdev_permissions pkcon_quit zypper_call);
+  qw(type_string_slow ensure_unlocked_desktop save_svirt_pty get_root_console_tty get_x11_console_tty ensure_serialdev_permissions pkcon_quit zypper_call disable_serial_getty);
```

Comment:
what is the limit for you to change it too simple `use utils` ? 40% of functions in file ? 80% ?

Reply:
Hehe, I'm backing up my solution with the legacy. In general it's more like I could not find any good solution where to put it and just put that to the related method. Considering how bit the utils file is, we save quite some resources by not importing all. Do you have a good suggestion for a separate module for this method and what to put there?